build/common.ngen.props

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0">
+  <ItemGroup Condition=" '$(SkipAssemblyAttributes)' != 'true'">
```

Comment:
can you please add some more information here?
It's kind of hard to understand what's going on here :)

Can this be done in a better way? A more extensible one rather than hardcoding all of it?

Reply:
I couldn't think of any better/ simple way to get this. Feel free to suggest any better approach. Besides, this should be one off thing (until we add new NuGet assembly as part of vsix project).